hydra-node/test/Hydra/ModelSpec.hs

```diff
@@ -118,12 +119,9 @@ prop_checkConflictFreeLiveness =
 prop_HydraModel :: Actions WorldState -> Property
 prop_HydraModel actions = property $
   runIOSimProp $ do
-    _ <- runActions runIt actions
+    _ <- runActions actions
     assert True
 
-runIt :: forall s. RunModel WorldState (StateT (Nodes (IOSim s)) (IOSim s))
-runIt = runModel
```

Comment:
If this is the only boilerplate we get in exchange for this `Realized`.. I'm sad.